buzsaki_lab_to_nwb/tingley_code_septal/convert_tingley_septal.py

```diff
@@ -1,66 +1,68 @@
 from pathlib import Path
+import sys
 import warnings
 
 from scipy.io import loadmat
 from nwb_conversion_tools.utils.json_schema import load_dict_from_file
 from nwb_conversion_tools.utils.json_schema import dict_deep_update
 
 from buzsaki_lab_to_nwb import TingleySeptalNWBConverter
+from joblib import Parallel, delayed
 
+n_jobs = 20
```

Comment:
Note that DANDIHub can support many more jobs than this if you're on a larger instance; to make the conversion go as fast as possible on run, you can max this out as long as `n_jobs * buffer_gb < max_RAM` (`max_RAM` is a DANDIHub instance value paired with the maximum number of cores allocated)